server/src/main/java/com/tencent/rss/server/storage/HdfsStorageManager.java

```diff
@@ -70,11 +71,11 @@ public Storage selectStorage(ShuffleDataReadEvent event) {
 
   @Override
   public void removeResources(String appId, Set<Integer> shuffleSet) {
-    Storage storage = getStorageByAppId(appId);
+    HdfsStorage storage = getStorageByAppId(appId);
     storage.removeHandlers(appId);
     appIdToStorages.remove(appId);
     ShuffleDeleteHandler deleteHandler = ShuffleHandlerFactory.getInstance()
-        .createShuffleDeleteHandler(new CreateShuffleDeleteHandlerRequest(StorageType.HDFS.name(), hadoopConf));
```

Comment:
Does hadoopConf in HdfsStorageManager still necessary? can it be removed?

Reply:
It may be better to use the same hadoop conf created and passed from upper layer.

Reply:
ok